config/protocol.unit_testnet.yml

```diff
@@ -1,4 +1,5 @@
 ProtocolConfiguration:
+  EnableStateRoot: true
```

Comment:
We can drop this, 3.0 is supposed to have some kind of state root by default anyway.

Reply:
Fixed.